lang/php/php.talon

```diff
@@ -0,0 +1,32 @@
+tag: user.php
+-
+tag(): user.code_imperative
+tag(): user.code_object_oriented
+tag(): user.code_libraries
+
+tag(): user.code_comment_line
+tag(): user.code_comment_block
+tag(): user.code_comment_documentation
+tag(): user.code_data_bool
+tag(): user.code_data_null
+tag(): user.code_operators_assignment
+tag(): user.code_operators_math
+tag(): user.code_functions
+
+settings():
+    user.code_private_function_formatter = "PRIVATE_CAMEL_CASE"
+    user.code_protected_function_formatter = "PRIVATE_CAMEL_CASE"
+    user.code_public_function_formatter = "PRIVATE_CAMEL_CASE"
+    user.code_private_variable_formatter = "PRIVATE_CAMEL_CASE"
+    user.code_protected_variable_formatter = "PRIVATE_CAMEL_CASE"
+    user.code_public_variable_formatter = "PRIVATE_CAMEL_CASE"
+
+(op | is) same: " === "
+(op | is) not same: " !== "
+
+state try: "try {\n"
+state catch: "catch (\\Throwable $exception) {\n"
+
+^var <phrase>$:
```

Comment:
Using anchors here interferes with chaining. You can probably drop them, allowing chaining from the start (so 'op plus var my variable' -> ' + $myVariable').

To make it chainable from the other end you might add a second copy of the command with the 'over' suffix like 'var <phrase> over'. With that I can do 'var my variable over op equals var my other variable over semicolon' and do the whole line in one utterance. That ability has the cost of not being able to end your variables with 'Over' as easily, but that's probably not a big deal.

Reply:
That's really clever, I didn't realize that the anchors interfered with chaining so much, I just saw them used in the functions or somewhere like that and borrowed them, but I love the idea of having an over variant.